server/views/temporary-accommodation/static/useNDelius.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = "Use nDelius" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
```

Comment:
I think in the GDS system you want rows, columns, then content, so this should contain something like:
`<div class="govuk-grid-column-two-thirds">`

Reply:
used `<div class="govuk-grid-column-full">`